sh.fhs.KatawaShoujoReEngineered.appdata.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>sh.fhs.KatawaShoujoReEngineered</id>
+    <metadata_license>CC-BY-SA-3.0</metadata_license>
+    <name>Katawa Shoujo: Re-Engineered</name>
+    <summary>A modern version of classic bishoujo-style visual novel</summary>
+    <description>
+        <p>
+            Katawa Shoujo is a bishoujo-style visual novel set in the fictional Yamaku High School for disabled children,
+            located somewhere in modern Japan. Hisao Nakai, a normal boy living a normal life, has his life turned upside
+            down when a congenital heart defect forces him to move to a new school after a long hospitalization. Despite his
+            difficulties, Hisao is able to find friends—and perhaps love, if he plays his cards right. There are five main
+            paths corresponding to the 5 main female characters, each path following the storyline pertaining to that character.
+        </p>
+        <p>
+            The story is told through the perspective of the main character, using a first person narrative. The game uses
+            a traditional text and sprite-based visual novel model with an ADV text box.
+        </p>
+        <p>
+            This is fanmade modernized version of classic visual novel. It is ported by Fleeting Heartbeat Studios to
+            Ren'Py 8, open sourced and de-obfuscated. It contains accessibility features, unavailable at the time of
+            initial game release, Russian and French localization out of the box, has useful function to enrich
+            gaming experience.
+        </p>
+    </description>
+    <categories>
+        <category>Game</category>
+        <category>AdventureGame</category>
+    </categories>
+    <url type="homepage">https://www.fhs.sh/projects</url>
+    <url type="faq">https://www.fhs.sh/about</url>
+    <launchable type="desktop-id">sh.fhs.KatawaShoujoReEngineered.desktop</launchable>
+    <releases>
+        <release version="1.3.0" date="2023-07-03">
+            <description>
+                <p>Katawa-Shoujo: Re-Engineered 1.3.0 contains many bugfixes since the previous release.</p>
+                <p>
+                    Full changelog available at
+                    <a href="https://codeberg.org/fhs/katawa-shoujo-re-engineered/releases">Codeberg</a>.
```

Comment:
Links are not allow here. There is a `url` element for the `release` tag that you can use.

Reply:
fxd